### PR TITLE
Bluebird

### PR DESCRIPTION
**Pull request by renujhamtani**
@sshumake @vrathee Please review

### CODE REVIEW

File: app/common/services/udsService.js [hunk 1]
```diff
@@ -5,237 +5,39 @@ angular.module('RedhatAccess.common').factory('udsService', [
     'RHAUtils',
     '$angularCacheFactory',
     function ($q, RHAUtils, $angularCacheFactory) {
-        function mapResponseObject(isCase,isComment,isEntitlement,isUser,isSolution,response) {
```
Comment (vrathee) [inline app/common/services/udsService.js:8]: Where are we doing this to the response now?

File: app/common/services/udsService.js [hunk 2]
```diff
@@ -5,237 +5,39 @@ angular.module('RedhatAccess.common').factory('udsService', [
...
-                        uds.fetchCaseDetails(
-                            function (response) {
-                                if (response.resource !== undefined) {
-                                    response=mapResponseObject(true,false,false,false,false,response);
-                                } else {
-                                    response=[];
-                                    deferred.reject("Unable to find case.");
-                                }
-                                var targetDate= RHAUtils.convertToTimezone(response.target_date_time);
-                                response.target_date = RHAUtils.formatDate(targetDate, 'MMM DD YYYY hh:mm:ss A Z');
```
Comment (vrathee) [inline app/common/services/udsService.js:197]: How will this be done now?

File: app/common/services/udsService.js [hunk 3]
```diff
@@ -246,382 +48,145 @@ angular.module('RedhatAccess.common').factory('udsService', [
...
                     get: function (caseNumber) {
-                        var deferred = $q.defer();
-                        uds.fetchCaseComments(
-                            function (response) {
-                                angular.forEach(response, angular.bind(this, function (comment) {
-                                    var lastModifiedDate = RHAUtils.convertToTimezone(comment.resource.lastModified);
-                                    var modifiedDate = comment.resource.lastModified;
-                                    comment.resource.sortModifiedDate = modifiedDate;
-                                    comment.resource.last_modified_date = RHAUtils.formatDate(lastModifiedDate, 'MMM DD YYYY');
-                                    comment.resource.last_modified_time = RHAUtils.formatDate(lastModifiedDate, 'hh:mm A Z');
```
Comment (vrathee) [inline app/common/services/udsService.js:273]: How will this be done now?

Comment (renujhamtani) [inline app/common/services/udsService.js:273]: Everything will be handled in the services now

**Pull request merged**